sdk/geofence/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-native-moengage-geofence",
-  "version": "4.0.1",
+  "version": "3.6.1",
```

Comment:
why is the version changed here? Is the branch not rebased with development?

Reply:
@umangmoe done